src/components/color-system/examples/colors-in-components.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-colors-in-components',
+    shadow: true,
+    styleUrl: 'colors-in-components.scss',
+})
+export class ColorExample {
```

Comment:
Should we put this example on a contrast-swatch backplate, like the other ones?

![image](https://user-images.githubusercontent.com/154725/91155384-109da700-e6c3-11ea-9599-4bdaf1b02198.png)

Reply:
yes, why not

Reply:
I've added a background, but it looks bad in dark mode. Can you take a look and adjust the styling?